src/components/ZodiacCard/ZodiacCard.css

```diff
@@ -0,0 +1,28 @@
+.zodiac {
+    padding: 100px;
+    color: var(--color1);
+    display: flex;
+    flex-direction: column;
+    align-items: auto;
+    justify-content: space-evenly;
+    text-align: center;
+    margin: 100px;
+  }
```

Comment:
If we reduce the margin here, and utilize flex-wrap: wrap; flex-direction: row; it may allow for the height of the app to be on a single page and the background would be less zoomed in.